etc/module.xml

```diff
@@ -4,6 +4,9 @@
     <module name="MageOS_ThemeAdminhtmlSwitcher">
         <sequence>
             <module name="Magento_Backend"/>
+            <module name="Magento_Theme"/>
+            <module name="Magento_Store"/>
+            <module name="Magento_Config"/>
```

Comment:
I always sort those by name, but this is of course totally optional :joy:

Reply:
@sprankhub, @rhoerr 
`Magento_Backend` depend on `Magento_Theme`, `Magento_Theme` depend on `Magento_Store`
I can only imaging to add `Magento_Config`

What would you say?

Reply:
Fair point, I'll adjust.